### PR TITLE
Add Developer Preview guidance to the SSG

### PR DESCRIPTION
**Pull request by aireilly**
A description of Developer Preview scope has been added in the customer portal: https://access.redhat.com/support/offerings/devpreview 

This PR adds the term "Developer Preview" to the SSG terms list and the Support section of the SSG. Related to https://github.com/redhat-documentation/supplementary-style-guide/issues/101

Preview: http://file.rdu.redhat.com/~ahoffer/2022/main-dev-preview.html#developer-preview-guidance

### CODE REVIEW

**Comment by leswilliams44** [comment]
This looks really good, @aireilly. Based on the following post on the CCS Google Chat earlier today, should <feature> in the admonition be replaced with a more descriptive term, such as the product or component name? 

An interesting call-out for Dev Preview from ^: NOTE: Individual features should NOT be considered Development Preview. Only releases of the "whole" product or component should use this terminology. This is in contrast to using "Technology Preview" to refer to individual features that are expected to be included in a future GA releases of a product.

**Comment by bergerhoffer** [comment]
> This looks really good, @aireilly. Just one question. Based on the following post on the CCS Google Chat earlier today, should _feature_ in the admonition be replaced with a more descriptive term, such as the product or component name?
> 
> An interesting call-out for Dev Preview from ^: NOTE: Individual features should NOT be considered Development Preview. Only releases of the "whole" product or component should use this terminology. This is in contrast to using "Technology Preview" to refer to individual features that are expected to be included in a future GA releases of a product.

@leswilliams44 According to the new https://access.redhat.com/support/offerings/devpreview page that they put together, it does list "technology, component, or feature". So I think it is probably okay to include. But perhaps we could rewrite the definition to more align with that page - basically expand that it could be a "technology, component, or feature". wdyt @aireilly?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:66]: Should we add "Development Preview" as incorrect? Or leave it off purposefully?

**Comment by aireilly** [comment]
Sounds good @leswilliams44 @bergerhoffer Need to clear a few things from plate before getting to this.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:62]: I would include that dev preview features are "no supported in any way by Red Hat" from the support scope statement.

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/support.adoc:14]: We might want to say something here about the possibility of it moving into TP status first.

**Review by bergerhoffer — COMMENTED** [review]
@aireilly some other feedback whenever you get a chance

**Comment by bergerhoffer** [comment]
Maybe one more thing we could consider referencing in the guidance sections for both TP and DP is the comparison sheet: https://access.redhat.com/articles/6966848. wdyt @aireilly?

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/style_guidelines/support.adoc:16]: Just FYI that PR #237 has been merged, whenever you're ready to pull in some of those changes

**Comment by aireilly** [comment]
> Maybe one more thing we could consider referencing in the guidance sections for both TP and DP is the comparison sheet: https://access.redhat.com/articles/6966848. wdyt @aireilly?

Sounds good! Need to get back to this. Day job taking over *sigh

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc
Comment (aireilly) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:66]: Added it :shrug:

**Comment by aireilly** [comment]
@bergerhoffer @leswilliams44 PTAL :)

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (aireilly) [reply supplementary_style_guide/style_guidelines/support.adoc]: Added something, but LMK if you think we should expand on it.

**Comment by bredamc** [comment]
@aireilly Is this PR ready for review? It's a little difficult to tell, when the "requested review" and "removed request" comments are combined :)

**Comment by aireilly** [comment]
Apologies - I did not see the repeated removed request notifications. This is indeed ready for review @bredamc

**Comment by bredamc** [comment]
Thanks for the prompt reply @aireilly! 

@leswilliams44 @bergerhoffer @roldanbob Are you available to review this PR?

**Review by bergerhoffer — DISMISSED** [review]
This looks great to me!

We need 3 SSG member approvals before merging, but I'm also wondering whether this is something we should run by others outside of the SSG for approval too. 

I just hesitate to add something like this without making sure that CCS doesn't have another group or team working on a similar effort. Or have strong objections to adding this at all. Maybe the content strategists would be a good group to reach out to about this?

**Comment by bergerhoffer** [comment]
I'm marking this as on hold, just as an indicator not to merge it before we check outside of the SSG group for any objections.

**Comment by aireilly** [comment]
@bergerhoffer Think this is ready to merge? I sent a mail to wider CS mailing list, didn't get any responses.

**Comment by roldanbob** [comment]
This LGTM! I'm assuming that Support and PM concur with the definition?

**Review by roldanbob — DISMISSED** [review]

**Comment by aireilly** [comment]
> This LGTM! I'm assuming that Support and PM concur with the definition?

This content was derived directly from the approved Support statement.

**Comment by bergerhoffer** [comment]
@aireilly Would you mind doing one more email on that CS thread (and CC me) and just say that if there is no feedback/objections this week, we will merge in this update next week.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc
Comment (bburt-rh) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:323]:
```suggestion
*Description*: Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. The software might not have any documentation, is subject to change or removal at any time, and testing is limited. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready.
```
- Not sure, but are we supposed to be adding `{nbsp}` between `Red` and `Hat` now?

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (bburt-rh) [inline supplementary_style_guide/style_guidelines/support.adoc:7]:
```suggestion
Developer Preview software provides early access to a technology, component, or feature, in advance of its possible inclusion in a Red Hat product offering. Customers can use Developer Preview software to test functionality and provide feedback during the development process. Documentation is not required for Developer Preview software, but if documentation is provided, it is subject to change or removal at any time. Also, testing is limited for Developer Preview software. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
```
- Same comment as above re `Red Hat` and non-breaking spaces.
- Same comment as above re anthropomorphism. Also "This" doesn't have a clear antecedent.
- I think this needs better logical flow. Consider my edits as one possible way to improve the flow.

Maybe consider just using the identical wording to the Glossary entry??

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:323]: I wouldn't worry about {nbsp} for now. We have issue #274 open to do a sweep through the guide to add as necessary.

As for the wording, some of it is used that way in the TP section too. So I might leave the first part the same, but the second part (adding "of it") isn't anywhere else, so you could potentially do that part.

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (bburt-rh) [inline supplementary_style_guide/style_guidelines/support.adoc:36]:
```suggestion
_<software_name>_ is Developer Preview software only. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview software for production or business-critical workloads. Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering. Customers can use this software to test functionality and provide feedback during the development process. This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
```
- See my similar comments above about anthropomorphism and non-breaking spaces with Red Hat.
- Reword the penultimate sentence so that the items listed in the series are grammatically parallel.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc
Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:323]: Nice work, @aireilly . 

I've some minor suggestions:

- Italize the first instance of _Developer Preview_. 
- Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready. > Red Hat does not support Developer Preview software, because the software is not functionally complete or production-ready.
- The software might not have any documentation, is subject to change or removal at any time, and has received limited testing. > The software might not include documentation, could undergo software changes or removal at any time, and might receive limited testing.

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (aireilly) [reply supplementary_style_guide/style_guidelines/support.adoc:7]: I went with your suggested edit thanks.

**Review by ndeevy — APPROVED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:313]: "early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering" is redundant. Recommend deleting "in advance of its possible inclusion in a Red Hat product offering".

**Review by bburt-rh — APPROVED** [review]

**Comment by bburt-rh** [comment]
@aireilly Thanks for the quick responses to my suggestions! This lgtm and I've approved it.

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (mportman12) [inline supplementary_style_guide/style_guidelines/support.adoc:36]: Ditto comment above.

**Review by mportman12 — COMMENTED** [review]
Left a couple of comments, otherwise looks good.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc
Comment (aireilly) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:313]: This isn't redundant. See https://access.redhat.com/support/offerings/devpreview

"Developer Preview versions are provided to expose features from upstream communities allowing early adopters and integrators to explore and interact with new capabilities in advance of their _possible inclusion_ in a Red Hat product offering."

File: supplementary_style_guide/style_guidelines/support.adoc
Comment (aireilly) [reply supplementary_style_guide/style_guidelines/support.adoc:36]: See above. Leaving this as is.

**Review by mportman12 — APPROVED** [review]
Ack, then LGTM

**Pull request merged**
